chipsec/modules/tools/secureboot/te.py

```diff
@@ -276,40 +275,40 @@ def IsValidPEHeader(data):
         #print "size < IMAGE_DOS_HEADER_size"
         return False
     signature, = struct.unpack("<H", data[:2])
-    if (signature != E_MAGIC):
+    if signature != E_MAGIC:
         #print "signature != E_MAGIC, 0x%04X != 0x%04X" % (E_MAGIC)
         return False
     e_lfanew, = struct.unpack("<I", data[IMAGE_DOS_HEADER_size - 4:IMAGE_DOS_HEADER_size])
-    if (e_lfanew >= size):
+    if e_lfanew >= size:
         #print "e_lfanew >= size"
         return False
-    if ((size - e_lfanew) < IMAGE_NT_HEADERS_size):
+    if (size - e_lfanew) < IMAGE_NT_HEADERS_size:
         #print "(size - e_lfanew) < IMAGE_NT_HEADERS_size"
         return False
     pe_signature, = struct.unpack("<I", data[e_lfanew:e_lfanew +4])
-    if (pe_signature != IMAGE_NT_SIGNATURE):
+    if pe_signature != IMAGE_NT_SIGNATURE:
         #print "pe_signature != IMAGE_NT_SIGNATURE"
         return False
     return True
 
 def replace_header(data):
-    if (not IsValidPEHeader(data)):
+    if not IsValidPEHeader(data):
         return None
     size = len(data)
     e_lfanew, = struct.unpack("<I", data[IMAGE_DOS_HEADER_size - 4:IMAGE_DOS_HEADER_size])
     #                          TimeDateStamp, PointerToSymbolTable, NumberOfSymbols, SizeOfOptionalHeader, Characteristics
     Machine, NumberOfSections, u1, u2, u3, SizeOfOptionalHeader, u5 \
      = struct.unpack(IMAGE_FILE_HEADER, data[e_lfanew +4:e_lfanew +4 +IMAGE_FILE_HEADER_size])
```

Comment:
```suggestion
     = struct.unpack(IMAGE_FILE_HEADER, data[e_lfanew + 4:e_lfanew + 4 + IMAGE_FILE_HEADER_size])
```
There are a few lines that could use spaces around the operators.